SAMLPortal/Middlewares/SetupAssistantMiddleware.cs

```diff
@@ -0,0 +1,51 @@
+using System.Linq;
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using SAMLPortal.Models;
+
+namespace SAMLPortal.Middlewares
+{
+	public class SetupAssistantMiddleware
+	{
+		private readonly RequestDelegate _next;
+
+		public SetupAssistantMiddleware(RequestDelegate next)
+		{
+			_next = next;
+		}
+
+		public Task Invoke(HttpContext httpContext)
+		{
+			var configurationStep = GlobalSettings.GetInt("CONFIG_SETUPASSISTANT_STEP");
+			string requestPath = httpContext.Request.Path.ToString().ToLower();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'requestPath'.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)